src/main/java/com/example/homework39/controllers/OrderController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.homework39.controllers;
+
+import com.example.homework39.entities.Order;
+import com.example.homework39.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+
+    public OrderController(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    @GetMapping("/{id}")
+    public Order getOrderById(@PathVariable Long id) {
+        return orderService.getOrderById(id);
+    }
+
+    @GetMapping("/")
+    public List<Order> getAllOrders() {
+        return orderService.getAllOrders();
+    }
+
+    @PostMapping("/")
+    public Order addOrder(@RequestBody Order order) {
+        return orderService.addOrder(order);
+    }
+
+    @DeleteMapping("/{id}")
+    public void deleteOrder(@PathVariable Long id) {
+        orderService.deleteOrder(id);

```

Comment:
Would be better to return some result of the deletion, so user is aware of the result of his request